packages/strapi-bookshelf/lib/index.js

```diff
@@ -117,6 +117,11 @@ module.exports = function(strapi) {
           _.forEach(models, (definition, model) => {
             globalName = _.upperFirst(_.camelCase(definition.globalId));
 
+            // Set the default values to model settings.
+            _.defaults(definition, {
+              primaryKey: 'id'
```

Comment:
It seems useless because Bookshelf already sets the `id` attribute as a default primary key (see http://bookshelfjs.org/docs/src_base_model.js.html#line115)